### ISSUE TITLE
Direct Messages

### ISSUE THREAD

**Issue by Jasmine2A** [bluesky-social/social-app#3668]
I was wondering if Bluesky will ever get direct messages. It is a staple to social media sites and people use it for private conversations and art commissions.

**Comment by surfdude29** [comment]
Yes, end-to-end encrypted direct messages are on the roadmap, see #1114.

**Comment by mschwendener** [comment]
(this could be _closed_ as duplicate)
